app/controllers/api/v1/bulk_download_controller.rb

```diff
@@ -280,30 +282,31 @@ def generate_curl_config
         # branch based on whether they provided a download_id, file_ids, or accessions
         if params[:download_id]
           download_req = DownloadRequest.find(params[:download_id])
-          if !download_req
```

Comment:
i think using `!download_req` produced slightly different results than `download_req.blank?` will (referencing these docs: https://apidock.com/rails/Object/blank%3F) is this logic shift intended?